tests/scancode/data/license/apache-1.1.txt

```diff
@@ -0,0 +1,58 @@
+/* ====================================================================

```

Comment:
Your tests are failing massively because you reused an existing test data directory by adding this file to `tests/scancode/data/license/`. Adding one file there makes another test break (`test_can_call_run_scan_as_a_function()`

So do not do that: create a new test data file directory solely for you unit test: otherwise you have tests that start have dependencies on each other and start failing for weird reasons.

Reply:
need a little bit time, I am currently figuring out git rebasing. Sorry.

Reply:
what do I need to do next?